Sources/Public/DotLottie/DotLottieFile.swift

```diff
@@ -25,14 +25,61 @@ public final class DotLottieFile {
     try decompress(data: data, to: fileUrl)
   }
 
-  // MARK: Internal
+  // MARK: Public
 
   /// Definition for a single animation within a `DotLottieFile`
-  struct Animation {
-    let animation: LottieAnimation
-    let configuration: DotLottieConfiguration
+  public struct Animation {
+    public let animation: LottieAnimation
+    public let configuration: DotLottieConfiguration
+  }
+
+  /// The `LottieAnimation` and `DotLottieConfiguration` for the given animation ID in this file
+  public func animation(for id: String? = nil) -> DotLottieFile.Animation? {
+    if let id = id {
+      return animations.first(where: { $0.configuration.id == id })
+    } else {
+      return animations.first
+    }
+  }
+
+  /// The `LottieAnimation` and `DotLottieConfiguration` for the given animation index in this file
+  public func animation(at index: Int) -> DotLottieFile.Animation? {
+    guard index < animations.count else { return nil }
+    return animations[index]
+  }
+
+  /// Returns the next animation in the array. Returns the first animation in case it's the last index
+  /// The next `LottieAnimation` and `DotLottieConfiguration`
+  public func nextAnimation(after animationId: String) -> DotLottieFile.Animation? {
```

Comment:
I wonder if it would make sense to just expose the `[Animation]` array publicly, rather than requiring consumers to use this sort of iterator API. If `animations: [Animation]` was public, then I don't think we'd need these `nextAnimation` and `previousAnimation` methods either. Thoughts?